src/Tickets/Seating/app/frontend/ticketsBlock/style.pcss

```diff
@@ -31,10 +31,44 @@
 	}
 }
 
+.tec-tickets-seating__modal--ar {
+	.tribe-tickets__tickets-item {
+		border-top: 0;
+	}
+
+	@media screen and (min-width:1024px) {
```

Comment:
can we re-use some of our defined media query vars maybe?
`event-tickets/common/src/resources/postcss/utilities/variables/_media-queries.pcss`

Reply:
I wasn't able to use them -_-